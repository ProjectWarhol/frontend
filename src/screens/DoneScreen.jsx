import React from 'react'
import { StyleSheet, Image, View } from 'react-native'
import { Text, Button } from 'react-native-elements'
import Spacer from '../components/Spacer'


const DoneScreen = ({navigation}) => {
    return (
    <View style={styles.container}>
    <Image source={require('../../assets/celebration.png')} style={styles.image}/>
    <Spacer/>
    <Text style={styles.text} h1>Your information is stored successfully!</Text>
    <Text h4>You can access it in setting</Text>
    <Spacer/>
    <Button title={'Proceed to UNOS'} onPress={()=>{navigation.navigate('mainFlow')}}/>
    </View>
    )
}

const styles = StyleSheet.create({
  container:{
    flexDirection: 'column',
    alignItems: 'center',
    justifyContent: 'center',
    height:'100%',
    width:'100%'
  },
  image:{
    width: 100,
    height: 100,
    resizeMode: 'contain'
  },
  text:{
    paddingBottom: 20
  }
})

export default DoneScreen
