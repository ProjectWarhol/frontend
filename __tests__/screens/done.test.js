import React from 'react'
import renderer from 'react-test-renderer'

import DoneScreen from '../../src/screens/DoneScreen'

jest.useFakeTimers()

test('renders correctly', () => {
    const tree = renderer.create(<DoneScreen />).toJSON()
    expect(tree).toMatchSnapshot()
})
