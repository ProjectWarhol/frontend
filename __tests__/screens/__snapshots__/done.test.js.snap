// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "column",
      "height": "100%",
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <Image
    source={1}
    style={
      Object {
        "height": 100,
        "resizeMode": "contain",
        "width": 100,
      }
    }
  />
  <View
    style={
      Object {
        "margin": 15,
      }
    }
  />
  <Text
    style={
      Object {
        "color": "#242424",
        "fontSize": 51.42857142857143,
        "paddingBottom": 20,
      }
    }
  >
    Great Job!
  </Text>
  <Text
    h5={true}
    style={
      Object {
        "color": "#242424",
      }
    }
  >
    you can always access your wallet information in the settings
  </Text>
  <View
    style={
      Object {
        "margin": 15,
      }
    }
  />
  <View
    style={
      Array [
        Object {
          "borderRadius": 3,
          "overflow": "hidden",
        },
        Object {
          "borderRadius": 3,
        },
        undefined,
        false,
      ]
    }
  >
    <View
      accessibilityRole="button"
      accessibilityState={
        Object {
          "busy": false,
          "disabled": false,
        }
      }
      accessible={true}
      focusable={true}
      onClick={[Function]}
      onResponderGrant={[Function]}
      onResponderMove={[Function]}
      onResponderRelease={[Function]}
      onResponderTerminate={[Function]}
      onResponderTerminationRequest={[Function]}
      onStartShouldSetResponder={[Function]}
      style={
        Object {
          "opacity": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#2089dc",
            "borderColor": "#2089dc",
            "borderRadius": 3,
            "borderWidth": 0,
            "flexDirection": "row",
            "justifyContent": "center",
            "padding": 8,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 18,
              "paddingVertical": 1,
              "textAlign": "center",
            }
          }
        >
          Proceed to UNOS
        </Text>
      </View>
    </View>
  </View>
</RCTSafeAreaView>
`;
